suite/tests/api/drdecode_aarch64.c

```diff
@@ -57,6 +57,41 @@ test_disasm(void)
         pc = disassemble_with_info(GD, pc, STDOUT, false /*no pc*/, true);
 }
 
+static void
+test_mov_instr_addr(void)
+{
+#if !defined(DR_HOST_NOT_TARGET)
+    instrlist_t *ilist = instrlist_create(GD);
+    instr_t *callee = INSTR_CREATE_label(GD);
+    instrlist_append(
+        ilist,
+        XINST_CREATE_move(GD, opnd_create_reg(DR_REG_X2), opnd_create_reg(DR_REG_X30)));
```

Comment:
I had to go look up what x30 is: how about using the alias DR_REG_LR which I think will make this easier to understand for those who forget that x30==lr

Reply:
Done.